test/testing/lock.cpp

```diff
@@ -27,7 +27,7 @@ is_locked(const fs::path& path)
     // From another process than the lockfile one, we can open/close
     // a new file descriptor without risk to clear locks
     int fd = open(path.c_str(), O_RDWR | O_CREAT, 0666);
-    bool is_locked = mamba::LockFile::is_locked(fd);
+    bool is_locked = mamba::Lock::is_locked(fd);
```

Comment:
I would like to keep the LockFile name -- because that's what it is. A "Lock" is a bit ambigous.